selinux/ipa.te

```diff
@@ -456,3 +447,14 @@ optional_policy(`
     kerberos_read_config(tomcat_t)
     kerberos_read_keytab(tomcat_t)
 ')
+
+optional_policy(`
```

Comment:
Could you please split this into two optional policy blocks? The types pki_tomcat_cert_t and node_t are provided by two different modules.

Reply:
Done, thanks